### DISCUSSION TITLE
Can I use it with a RPi 3 or 4?

### DISCUSSION THREAD

**Discussion by 256FF** [fhdm-dev/scoppy#10] [Q&A] [answered]
I've some RPis 3 and 4 lying around. Can I use the image with this too?

**Comment by fhdm-dev** [comment, marked as answer]
Unfortunately not. It only works with the RPi Pico in combination with an Android phone/tablet.

**Comment by santlaha** [comment]
> I've some RPis 3 and 4 lying around. Can I use the image with this too?

code is rp2040 hardware specific. As the fastest method to poll ADC values in, is using DMA and state machines.

**Comment by santlaha** [comment]
> I've some RPis 3 and 4 lying around. Can I use the image with this too?

code is rp2040 hardware specific. As the fastest method to poll ADC values in, is using DMA and state machines.
